scripts/content.js

```diff
@@ -11,6 +11,7 @@ const getValuesFromStorage = async () => {
   chrome.storage.local.get(["highlightWords", "ignoreWords"]).then((result) => {
     values = result;
     applyHighlights(values);
+    removeUnhighlightedComments();    
```

Comment:
Is this function call in the right place? There are two places where we're calling `applyHighlights`: here, and in the listener for when values are changed. Should this also be called in the listener as well?

Reply:
Oops - thanks for catching this.  I added the missing function call in the listener.